contrib/forms/leg_length/report.php

```diff
@@ -112,6 +187,7 @@ function leg_length_report($pid, $encounter, $cols, $id)
             </td>
             <td>
 
+
                 <p><?php echo $data["UMB_left"]?>&nbsp;</p>
```

Comment:
Seems really odd to be adding this code in this script. What this meant to be changed?